tests/kernel/install_ltp.pm

```diff
@@ -32,7 +32,7 @@ sub add_we_repo_if_available {
     return if check_var('DISTRI', 'opensuse');
 
     my $ar_url;
-    my $we_repo = get_var('REPO_SLE_WE15_POOL');
+    my $we_repo = get_var('REPO_SLE_WE_POOL');
```

Comment:
@pevik could you please take a look. This won't work with SLE 15, as there it's called REPO_SLE-PRODUCT-WE is it fine to keep it like that?